From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dmitrii <bankersenator@gmail.com>
Date: Sun, 14 Apr 2024 17:59:03 +0000
Subject: [PATCH 058/226] AutofillManagerService: silence spam

04-14 23:30:17.922  1748  1833 W AutofillManagerService: Ignoring not allowed compat package org.chromium.chrome
04-14 23:30:17.922  1748  1833 W AutofillManagerService: Ignoring not allowed compat package org.bromite.chromium

Signed-off-by: Dmitrii <bankersenator@gmail.com>
---
 .../com/android/server/autofill/AutofillManagerService.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/autofill/java/com/android/server/autofill/AutofillManagerService.java b/services/autofill/java/com/android/server/autofill/AutofillManagerService.java
index 3d56625dc9..8de7ba08c5 100644
--- a/services/autofill/java/com/android/server/autofill/AutofillManagerService.java
+++ b/services/autofill/java/com/android/server/autofill/AutofillManagerService.java
@@ -902,7 +902,7 @@ public final class AutofillManagerService
         for (int i = 0; i < compatPackageCount; i++) {
             final String packageName = compatPackages.keyAt(i);
             if (allowedPackages == null || !allowedPackages.containsKey(packageName)) {
-                Slog.w(TAG, "Ignoring not allowed compat package " + packageName);
+                if (sDebug) Slog.w(TAG, "Ignoring not allowed compat package " + packageName);
                 continue;
             }
             final Long maxVersionCode = compatPackages.valueAt(i);
